core-bundle/src/Resources/contao/controllers/FrontendIndex.php

```diff
@@ -232,11 +234,11 @@ public function renderPage($pageModel)
 		// Set the admin e-mail address
 		if ($objPage->adminEmail)
 		{
-			list($GLOBALS['TL_ADMIN_NAME'], $GLOBALS['TL_ADMIN_EMAIL']) = StringUtil::splitFriendlyEmail($objPage->adminEmail);
+			[$GLOBALS['TL_ADMIN_NAME'], $GLOBALS['TL_ADMIN_EMAIL']] = StringUtil::splitFriendlyEmail($objPage->adminEmail);
```

Comment:
These changes seem unrelated?

Reply:
Fixed in 0eb40934e9773ee44d8604d25570bf918bc8e290. Somehow my IDE started auto-fixing this stuff recently 🙄